sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/ReactorNettyClientProvider.java

```diff
@@ -0,0 +1,12 @@
+package com.azure.core.http.netty;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.implementation.http.spi.HttpClientProvider;
+
+public class ReactorNettyClientProvider implements HttpClientProvider {
+
+    @Override
+    public HttpClient createNewInstance() {
```

Comment:
https://github.com/Azure/azure-sdk-for-java/pull/4572/files#diff-4abe8da7b04f10453d6c969e4eadcc0dR29
the method is called `createInstance()`. Iprefer that to `createNewInstance()`. I feel like 'new' is implied when you create it. 🤔